lib/css/components/_stacks-progress-bars.less

```diff
@@ -215,3 +215,56 @@
         }
     }
 }
+
+//  ===========================================================================
+//  $   CIRCULAR
+//  ---------------------------------------------------------------------------
+.s-progress__circular {
+    --s-progress-value: 0;
+    --s-circle-circumference: 88; // 2πr, r = 14.
+
+    background: none;
+    width: @su32;
+    height: @su32;
+
+    .s-progress-bar {
+        // Make everything originate from the top of the circle
+        transform: rotate(270deg);
+
+        circle {
+            stroke-width: 4;
+            fill: none;
+            stroke-linecap: round;
+        }
+
+        circle:nth-of-type(1) {
+            stroke: currentColor;
+            opacity: .4;
+        }
+
+        circle:nth-of-type(2) {
+            stroke: currentColor;
+            stroke-dasharray: var(--s-circle-circumference);
+
+            // Multiply the circle circumference by an the opposite percentage of what we want to display
+            // For example 70%, represented as a decimal is 0.7
+            // The opposite of that is 0.3 so ( 1 - 0.7 )
+            stroke-dashoffset: calc( ((1 - var(--s-progress-value)) * var(--s-circle-circumference)) * 1px); // Multiply everything by 1px since Safari and Firefox require these to be in pixels
+        }
+    }
```

Comment:
I figured out why 5% looks so much larger than 95%. The [round linecap](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/stroke-linecap#round) will

> be extended by a half circle with a diameter equal to the stroke width

To compensate for this, I moved the stroke width out into a variable and re-added it in the final calculation to compensate. Additionally, I moved the pi calculation into less as I suggested above.

![image](https://user-images.githubusercontent.com/14169651/108116704-fb01ac80-7069-11eb-98f8-2cd042b6c86e.png)


```suggestion
.s-progress__circular {
    --s-progress-value: 0;
    @circleCircumference: (2 * pi() * 14); // 2πr, r = 14.

    background: none;
    width: @su32;
    height: @su32;

    .s-progress-bar {
        // Make everything originate from the top of the circle
        transform: rotate(270deg);

        @strokeWidth: 4;
        circle {
            stroke-width: @strokeWidth;
            fill: none;
            stroke-linecap: round;
        }

        circle:nth-of-type(1) {
            stroke: currentColor;
            opacity: .4;
        }

        circle:nth-of-type(2) {
            stroke: currentColor;
            stroke-dasharray: @circleCircumference;

            // Multiply the circle circumference by an the opposite percentage of what we want to display
            // For example 70%, represented as a decimal is 0.7
            // The opposite of that is 0.3 so ( 1 - 0.7 )
            // Additionally, since the "round" linecap adds (width / 2)px length, adjust the total offset to compensate
            stroke-dashoffset: calc( (((1 - var(--s-progress-value)) * @circleCircumference) + @strokeWidth) * 1px); // Multiply everything by 1px since Safari and Firefox require these to be in pixels
        }
    }
```